ludwig/models/modules/sequence_decoders.py

```diff
@@ -149,104 +149,56 @@ class SequenceTaggerDecoder(Layer):
     def __init__(
             self,
             initializer=None,
```

Comment:
this doesn't seem to be used, can remove

Reply:
done